src/pages/kwonojae/Login/Login.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import './Login.scss';
+import { Link } from 'react-router-dom';
+import { withRouter } from 'react-router-dom';
+
+class LoginOJ extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      id: '',
+      pw: '',
+    };
+  }
+
+  handleInput = e => {
+    const { name, value } = e.target;
+
+    this.setState({ [name]: value });
+  };
+
+  validation = () => {
+    const { id, pw } = this.state;
+    const isValidId = id.includes('@');
+    const isValidPw = pw.length > 4;
+
+    return isValidId && isValidPw;
```

Comment:
```suggestion
const validIdAndPw = id.includes('@') && pw.length > 4
return validIdAndPw
```
개별적으로 비교하는 것이 아니라면, 위와 같이 수정해도 좋을 것 같습니다!

Reply:
확인하고 수정하였습니다~